README.md

```diff
@@ -726,6 +654,33 @@ Below are listed all the variables you can customize (you may also want to take
     postfix_smtpd_relay_restrictions_override: permit_mynetworks
     # allows overriding the default duration for keeping notifications
     openwisp2_notifications_delete_old_notifications: 10
+    openwisp2_users_auth_api: true
+    openwisp2_radius_sms_backend: "sendsms.backends.console.SmsBackend"
+    openwisp2_radius_sms_token_max_ip_daily: 25
+    openwisp2_freeradius_allowed_hosts: ["127.0.0.1"]
+    freeradius_dir: /etc/freeradius/3.0
+    freeradius_mods_available_dir: "{{ freeradius_dir }}/mods-available"
+    freeradius_mods_enabled_dir: "{{ freeradius_dir }}/mods-enabled"
+    freeradius_sites_available_dir: "{{ freeradius_dir }}/sites-available"
+    freeradius_sites_enabled_dir: "{{ freeradius_dir }}/sites-enabled"
+    freeradius_sql:
+        driver: rlm_sql_sqlite
+        dialect: sqlite
+        host: ""
+        port: ""
+        dbname: ""
+        user: ""
+        password: ""
+    freeradius_rest:
+        url: "https://{{ inventory_hostname }}/api/v1/freeradius"
+    freeradius_clients_ip: "0.0.0.0/0"
+    freeradius_clients_key: "admin"
```

Comment:
@nemesisdesign are these defaults fine or can you suggest between defaults for `client.conf`?

Reply:
replied regarding clients elsewhere